src/analyzer.ts

```diff
@@ -98,6 +98,10 @@ export class Analyzer {
     return this._context.load(resolvedUrl, providedContents);
   }
 
+  canResolveUrl(url: string): string {
```

Comment:
Does the bundler need this? It won't even see these imports in the Documents. Or does it need this for when it walks the AST itself.